frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoViewModel.java

```diff
@@ -119,8 +119,16 @@ public void updateDeviceList() {
             Iterator<UsbDevice> deviceIterator = deviceList.values().iterator();
             while (deviceIterator.hasNext()){
                 UsbDevice device = deviceIterator.next();
-                this.device = new GoDeviceInfo(device);
-                break; // only one device supported for now
+                // BitBox02 Vendor ID: 0x03eb, Product ID: 0x2403.
```

Comment:
So the device_filter.xml does not make sure that the deviceList only has devices with right vendor/product id? wtf?

anyway, make a comment in device_filter.xml that any changes have to be also done here?

Reply:
device_filter.xml is to launch the app when it's still closed and you plug it the device first.

This one, updateDeviceList, is when the app is already launched and opened, and you plug in a device after. Well, it also runs when device_filter.xml is trigger but in this case there's no issue becaues device_filter.xml already filters by the known vendor/product ID.

Makes sense?

Reply:
Slightly updated the comment. Hope it makes it better.